tower_cli/cli/action.py

```diff
@@ -14,6 +14,9 @@
 # limitations under the License.
 
 import click
+from click.formatting import join_options
+
+from tower_cli.conf import SETTINGS_PARMS
```

Comment:
This is a really slick way to implement this feature. I was wondering how you got this working with so little code...